specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/examples/compute/GetVirtualMachineScaleSetWithUserData.json

```diff
@@ -99,7 +99,8 @@
           "platformFaultDomainCount": 1,
           "hostGroup": {
             "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/hostGroups/myHostGroup"
-          }
+          },
+          "timeCreated": "2021-06-27T01:02:38.3138469+00:00"
```

Comment:
this example file is for for the specific userData scenario, don;t we have a basic example file like GetVirtualMachineScaleSet.json where we can add this property, as this property will be present now in every resource response.

Reply:
Oddly no, should I create it? For examples of the flavor GetVMSS there are:
| Mode   | LastWriteTime      | Length Name | Name                                                                       |
|--------|--------------------|-------------|----------------------------------------------------------------------------|
| -a---- | 11/8/2021 10:38 AM | 3307        | GetVirtualMachineScaleSetAutoPlacedOnDedicatedHostGroup.json               |
| -a---- | 11/8/2021 10:38 AM | 956         | GetVirtualMachineScaleSetVMExtensions.json                                 |
| -a---- | 11/8/2021 10:38 AM | 2001        | GetVirtualMachineScaleSetVMInstanceViewAutoPlacedOnDedicatedHostGroup.json |
| -a---- | 11/8/2021 10:38 AM | 5183        | GetVirtualMachineScaleSetVMWithUserData.json                               |
| -a---- | 11/8/2021 10:38 AM | 4272        | GetVirtualMachineScaleSetWithUserData.json                                 |

Reply:
We can use GetVirtualMachineScaleSetVMExtensions which should be the basic api model in case of VMSS.

Reply:
The GetVirtualMachineScaleSetVMExtensions example expects an extension object based on the properties (below), not a VMSS object. 

GetVirtualMachineScaleSetWithUserData and GetVirtualMachineScaleSetAutoPlacedOnDedicatedHostGroup looks like the 2 Get examples with VMSS objects. 

The other 2 look like they expect VM objects.

```
{
  "parameters": {
    "subscriptionId": "{subscription-id}",
    "resourceGroupName": "myResourceGroup",
    "vmScaleSetName": "myvmScaleSet",
    "instanceId": "0",
    "vmExtensionName": "myVMExtension",
    "api-version": "2021-11-01"
  },
  "responses": {
    "200": {
      "body": {
        "properties": {
          "autoUpgradeMinorVersion": true,
          "provisioningState": "Creating",
          "publisher": "extPublisher",
          "type": "extType",
          "typeHandlerVersion": "1.2",
          "settings": {
            "UserName": "xyz@microsoft.com"
          }
        },
        "name": "myVMExtension",
        "type": "Microsoft.Compute/virtualMachineScaleSets/virtualMachines/extensions",
        "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachineScaleSets/myvmScaleSet/virtualMachines/0/extensions/myVMExtension"
      }
    }
  }
}
```

Reply:
Oks, then let us create the basic example file for GetVMSS like how we have it for GetVirtualMachine.